docs/dev/04-public-api.md

```diff
@@ -111,6 +111,22 @@ runner.run({port: 9876}, function(exitCode) {
 })
 ```
 
+## karma.stopper
+
+### **stopper.stop(options, [callback=process.exit])**
+
+This function will signal a running server to stop.  Equivalent of `karma stop`.  
```

Comment:
We are doing sth very similar here: https://github.com/karma-runner/grunt-karma/blob/master/tasks/grunt-karma.js#L114-L134 so it would be nice if we could replace that with this new option, rather than duplication that effort